src/bo4e_cli/generate/python/format.py

```diff
@@ -0,0 +1,71 @@
+"""
+Contains functions to post process Python code using black, isort and autoflake.
+"""
+
+import os
+import subprocess
+import tempfile
+from pathlib import Path
+
+from datamodel_code_generator import PythonVersion
+from datamodel_code_generator.format import CodeFormatter
+from rich.progress import track
+
+from bo4e_cli.io.console import CONSOLE
+
+
+def remove_unused_imports(code: str) -> str:
+    """
+    Removes unused imports from the given code using autoflake.
+    """
+    # Create a temporary file
+    with tempfile.NamedTemporaryFile(suffix=".py", delete=False) as tmp_file:
+        tmp_file_name = tmp_file.name
+        tmp_file.write(code.encode("utf-8"))
+
+    # Run autoflake to remove unused imports
+    subprocess.run(["autoflake", "--remove-all-unused-imports", "--in-place", tmp_file_name], check=True)
+
+    # Read the cleaned code from the temporary file
+    with open(tmp_file_name, "r", encoding="utf-8") as tmp_file:
+        cleaned_code = tmp_file.read()
+
+    # Clean up the temporary file
+    os.remove(tmp_file_name)
```

Comment:
nur aus Neugier, gibt es in pathlib nichts um eine Datei zu löschen?

Reply:
Doch klar, aber das war mit dem Konstrukt mit diesen Temporary files n bissl schwierig. Hätte natürlich auch `Path(tmp_file_name).unlink()` machen können.